PLEP-0007.rst

```diff
@@ -0,0 +1,847 @@
+isolate the essential physics required to model collisionless magnetic
+reconnection (Birn2001_). The strategy was to perform simulations with
+the same ICs and BCs using multiple codes with different physical
+models, ranging from resistive MHD to fully electromagnetic PIC.  A
+scientist decides to attempt to reproduce the results of the GEM
+challenge using a larger computational domain and higher resolution.
+
+.. _testPIC:
+
+Testing PIC algorithms
+----------------------
+
+Different PIC algorithms and particle pushers offer different
+advantages and are needed for different situations. A researcher
+decides to test different algorithms for a variety of different
+physical situations.
+
+.. _WholeDevice:
+
+Integrated tokamak modeling
+---------------------------
+
+A major goal of magnetic fusion energy science is to create predictive
+whole-device models for magnetic confinement devices such as tokamaks
+(Poli2018_). Whole-device models will be used by scientists who wish
+to understand fundamental plasma behavior, engineers who wish to
+design new devices, and control room operators who are overseeing
+discharges. Different physical processes will be important in
+different parts of the device and at different time and length scales.
+The model would need to include a variety of effects including but not
+limited to plasma-wall interactions, radio frequency heating, and
+neutral beam injection.  Simulation results will need to be tested
+against multiple plasma diagnostics.
+
+.. [Poli2018] F. M. Poli (2018), `*Integrated Tokamak modeling: When
+   physics informs engineering and research planning*
+   <https://doi.org/10.1063/1.5021489>`_, Physics of Plasmas, 25,
+   055602, doi: 10.1063/1.5021489
+
+Modeling the solar chromosphere
+-------------------------------
+
+The solar chromosphere hosts a rich variety of physical processes.
+Many of the simplifying assumptions that are valid in either the
+photosphere below or the corona above cannot be applied to the
+chromosphere [e.g., the assumption of local thermodynamic equilibrium
+(LTE)].  The plasma is partially ionized, so interactions between
+neutrals and charged particles are important.  Non-LTE radiative
+transfer is required.  Shocks contribute to the heating.  Some solar
+physicists will want to model a beam of energetic electrons
+precipitating from the corona into the chromosphere during solar
+flares. Synthetic observables will be required to validate simulation
+results against reality.
+
+Software Requirements Specification
+===================================
+
+Availability requirements
+-------------------------
+
+* Develop openly under the Open Source Initiative approved `BSD+Patent
+  <https://opensource.org/licenses/BSDplusPatent>`_ license.
+
+* Allow installation in a single line with widely-used package
+  managers.
+
+  - Do not require users to compile external libraries, edit make
+    files, change compiler flags, or run bash scripts.
```

Comment:
"... unless it helps performance" for the HPC-minded? This is a pretty strict requirement. I fully understand why it's here, though - I'm just pointing out a possible point of contention.